services/graph/pkg/service/v0/users.go

```diff
@@ -885,20 +884,26 @@ func (g Graph) patchUser(w http.ResponseWriter, r *http.Request, nameOrID string
 		errorcode.RenderError(w, r, err)
 		return
 	}
+
 	u.PreferredLanguage = preferredLanguage
+	g.patchUserResponse(w, r, u, nameOrID, features)
```

Comment:
This is not a new issue but I guess the events should always contain the userid (never the name) and `nameOrID` can indeed be a username. We should probably use u.GetID() for the event. I guess you can just leave out the `nameOrID` parameter. And have fill the id from the user object in `patchUserResponse`.

Looks good to me otherwise.

Reply:
Good point :+1: Did as requested.